src/badguy/badguy.cpp

```diff
@@ -645,11 +645,15 @@ BadGuy::kill_fall()
 void
 BadGuy::run_dead_script()
 {
+  if(m_is_active_flag) {
```

Comment:
If this does end up staying, please invert the control flow.

Reply:
I can do that soon, whats the purpose tho?

Reply:
Satisfies my OCD.

Reply:
Bruh

Reply:
```cpp
if(!m_active_flag)
   return;

m_active_flag = false;
```

is easier to read IMO.

Reply:
i did the thing maybe

Reply:
> ```c++
> if(!m_active_flag)
>    return;
> 
> m_active_flag = false;
> ```
> 
> is easier to read IMO.

somehow this did not load on either the Discord or the Github, give me one more second